### PR TITLE
Improve handling of wikiDomains and Allowed parameters

### PR DESCRIPTION
**Pull request by paul90**
* make wikiDomains and Allowed really optional
* add warning if neither wikiDomains or Allowed are configured
* add warning if using default security module (read-only)

### CODE REVIEW

**Comment by paul90** [comment]
Part of a solution to #87, here we make the use of wikiDomains and allowed parameters truly optional.

File: farm.coffee [hunk 1]
```diff
@@ -25,9 +25,9 @@ module.exports = exports = (argv) ->
   runningServers = []
 
   if argv.allowed
-    allowedHosts = argv.allowed.split(',')
+    allowedHosts = _.split(argv.allowed, ',')
```
Comment (WardCunningham) [inline farm.coffee]: Just curious. Why the move away from native split?

**Comment by WardCunningham** [comment]
Thanks for this. We are accumulating special cases but these seem necessary. The log notices are a welcome addition.

**Comment by paul90** [comment]
Published as + wiki@0.10.1

File: farm.coffee (hunk 1 above)
Comment (paul90) [inline farm.coffee]: The code was crashing, with the trace pointing to it as the problem.

**Pull request merged**